src/libs/Middleware/SaveResponseInOnyx.js

```diff
@@ -13,23 +15,27 @@ function SaveResponseInOnyx(response, request) {
                 return;
             }
 
+            // For most requests we can immediately update Onyx. For write requests we queue the updates and apply them after the sequential queue has flushed to prevent a replay effect in
+            // the UI. See https://github.com/Expensify/App/issues/12775 for more info.
```

Comment:
fwiw it is easy for people to find linked issues by using git blame. usually there is no need to add a link to a github issue.